internal/daemon/api_services.go

```diff
@@ -20,16 +20,18 @@ import (
 	"net/http"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/canonical/pebble/internal/overlord/servstate"
 	"github.com/canonical/pebble/internal/overlord/state"
 	"github.com/canonical/pebble/internal/strutil"
 )
 
 type serviceInfo struct {
-	Name    string `json:"name"`
-	Startup string `json:"startup"`
-	Current string `json:"current"`
+	Name         string     `json:"name"`
+	Startup      string     `json:"startup"`
+	Current      string     `json:"current"`
+	CurrentSince *time.Time `json:"current-since,omitempty"` // pointer as omitempty doesn't work with time.Time directly
```

Comment:
That's awkward, considering times have IsZero which fits properly with the notion of omitempty.

This is an internal detail, so I'm not blocking on it, but it's curious that the standard json package is misbehaving there.